src/lib.rs

```diff
@@ -89,7 +89,10 @@ pub use indexmap::{
 pub use indexset::{FnvIndexSet, IndexSet, Iter as IndexSetIter};
 pub use linear_map::LinearMap;
 pub use string::String;
-pub use vec::Vec;
+#[cfg(doc)]
+#[doc(hidden)]
+pub use vec::VecInner as _;
+pub use vec::{Vec, VecView};
```

Comment:
This is not required any-more with the latest nightly, but is needed to work with stable.

Reply:
By “this” you mean only the `doc(hidden)` workaround? If so, does it matter given `docs.rs` uses nightly anyways?

Reply:
Yes this is what I mean. `docs.rs` uses nightly, but the workaround might still be useful for people using `cargo doc` locally (I often do so that the search covers all my dependencies).

Reply:
I guess it doesn't hurt, but let's add a comment why `doc(hidden)` is needed.

Reply:
I added a comment with a link to the rustdoc issue that this works around and with justification for the `#[doc(hidden)]` and the `#[cfg(doc)]`.